pydantic/_migration.py

```diff
@@ -288,7 +288,7 @@ def wrapper(name: str) -> object:
         if import_path == 'pydantic:BaseSettings':
             raise PydanticImportError(
                 '`BaseSettings` has been moved to the `pydantic-settings` package. '
-                f'See https://docs.pydantic.dev/{VERSION}/migration/#basesettings-has-moved-to-pydantic-settings '
+                f'See https://docs.pydantic.dev/latest/migration/#basesettings-has-moved-to-pydantic-settings '
```

Comment:
It would be fixed by removing the `minor` version number.
https://docs.pydantic.dev/2.1/migration/#basesettings-has-moved-to-pydantic-settings would work.

```suggestion
                f'See https://docs.pydantic.dev/{".".join(VERSION.split(".")[:2])}/migration/#basesettings-has-moved-to-pydantic-settings '
```